index.rst

```diff
@@ -711,6 +711,33 @@ Getter name inference works the same way (of course, replacing 'set' with 'get')
 
 .. _composite-policy:
 
+Component Subset
+^^^^^^^^^^^^^^^^
+
+If, instead of a single component object, a list of component objects is desired
+for a given component, the keyword 'subset' can be added to the component
+dataset definition, with the value ``True``.
+
+.. code-block:: none
+
+    <datasetType name>: {
+        composite: {
+            <component name>: {
+                subset: True
+                ...
+            }
+            ...
+        }
+    }
+
+When butler is getting components for a comosite and sees this keyword, it will
+use Butler.subset to find the component (instead of butler.get, which is the
```

Comment:
Consistent case: "butler" or "Butler".  Even better, have these be references to the method documentation.